src/main/java/com/salesforce/dataloader/security/EncryptionUtil.java

```diff
@@ -270,12 +276,16 @@ public void setCipherKeyFromFilePath(String keyFilename) throws IOException {
     private static void printUsage() {
         LOGGER.info("\nUtility to encrypt a string based on a static or a provided key");
         LOGGER.info("Options (mutually exclusive - use one at a time): \n"
-                + "\t-g <seed text>                                 Generate key based on seed\n"
-                + "\t-v <encrypted> <decrypted value> [Path to Key] Validate whether decryption of encrypted value matches the decrypted value, optionally provide key file\n"
-                + "\t-e <plain text> [Path to Key]                  Encrypt a plain text value, optionally provide key file (generate key using option -g)");
+                + "To be deprecated:"
+                + "\t\t <seed text>                                 Generate key based on seed\n"
+                + "\t\t<encrypted> <decrypted value> [Path to Key] Validate whether decryption of encrypted value matches the decrypted value, optionally provide key file\n"
+                + "\t\t-e <plain text> [Path to Key]                  Encrypt a plain text value, optionally provide key file (generate key using option -g\n)"
```

Comment:
Remove all support for generating DES keys (especially since the method used isn't even a secure way to generate any kind of key), and for encrypting using DES. Either replace these functions with a migrate utility (though that should just happen automatically, instead) or make them use AES instead (and don't generate the AES keys from a password, do it using raw output of `SecureRandom` and do it automatically).